packages/waku/src/router/define-router.ts

```diff
@@ -101,7 +101,7 @@ export function unstable_defineRouter(fns: {
   handleApi?: (
     path: string,
     options: {
-      body: ReadableStream | null;
+      body?: ReadableStream | null;
```

Comment:
We always pass body, so this doesn't have to be optional.
```suggestion
      body: ReadableStream | null;
```

Reply:
Hmm, this was for tests. Can you fix the test somehow?